### ISSUE TITLE
Sprockets method `register_engine` is deprecated

### ISSUE THREAD

**Issue by mattfelten** [jekyll/jekyll-assets#313]
```
$ bundle exec jekyll build --trace
Configuration file: ....../_config.yml
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from block in <top (required)> at ~/.rvm/gems/ruby-2.3.0/gems/jekyll-assets-2.2.7/lib/jekyll/assets/processors/liquid.rb:35)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from block in <top (required)> at ~/.rvm/gems/ruby-2.3.0/gems/jekyll-assets-2.2.7/lib/jekyll/assets/processors/liquid.rb:35)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from block in <top (required)> at ~/.rvm/gems/ruby-2.3.0/gems/jekyll-assets-2.2.7/lib/jekyll/assets/processors/liquid.rb:35)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from block in <top (required)> at ~/.rvm/gems/ruby-2.3.0/gems/jekyll-assets-2.2.7/lib/jekyll/assets/processors/liquid.rb:35)
            Source: ........
       Destination: ....../_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
DEPRECATION WARNING: You are using the a deprecated processor interface #<Proc:0x007fa21503b718@~/.rvm/gems/ruby-2.3.0/gems/autoprefixer-rails-6.3.7/lib/autoprefixer-rails/sprockets.rb:37>.
Please update your processor interface:
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from install at ~/.rvm/gems/ruby-2.3.0/gems/autoprefixer-rails-6.3.7/lib/autoprefixer-rails/sprockets.rb:37)
                    done in 1.874 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
```

How do I stop all these depreciation warnings?

**Comment by envygeeks** [comment]
@mattfelten this issue is already addressed on master and a release will happen tomorrow.

**Comment by mattfelten** [comment]
Cool thanks. I searched but only (mistakenly) saw that other topic.

**Comment by mattfelten** [comment]
Any update on that release?

**Comment by anroots** [comment]
This was "fixed" in release `2.2.8` by downgrading Sprockets from `3.7.0` to `3.6.3`.

```bash
$ bundle update jekyll-assets 
Installing sprockets 3.6.3 (was 3.7.0)
Installing jekyll-assets 2.2.8 (was 2.2.7)
```

Relevant commit: https://github.com/jekyll/jekyll-assets/commit/da00c80bdbdd8e96b16123777bd38c422c94ac20. No DPERECATION warnings appear in my console any more.
